doc/stdlib/math.rst

```diff
@@ -1966,62 +3013,23 @@ some built-in integration support in Julia.
 
 .. function:: quadgk(f, a,b,c...; reltol=sqrt(eps), abstol=0, maxevals=10^7, order=7, norm=vecnorm)
 
-   Numerically integrate the function ``f(x)`` from ``a`` to ``b``,
-   and optionally over additional intervals ``b`` to ``c`` and so on.
-   Keyword options include a relative error tolerance ``reltol`` (defaults
-   to ``sqrt(eps)`` in the precision of the endpoints), an absolute error
-   tolerance ``abstol`` (defaults to 0), a maximum number of function
-   evaluations ``maxevals`` (defaults to ``10^7``), and the ``order``
-   of the integration rule (defaults to 7).
-
-   Returns a pair ``(I,E)`` of the estimated integral ``I`` and an
-   estimated upper bound on the absolute error ``E``.  If ``maxevals``
-   is not exceeded then ``E <= max(abstol, reltol*norm(I))`` will hold.
-   (Note that it is useful to specify a positive ``abstol`` in cases where
-   ``norm(I)`` may be zero.)
-
-   The endpoints ``a`` etcetera can also be complex (in which case the
-   integral is performed over straight-line segments in the complex
-   plane).  If the endpoints are ``BigFloat``, then the integration
-   will be performed in ``BigFloat`` precision as well (note: it is
-   advisable to increase the integration ``order`` in rough proportion
-   to the precision, for smooth integrands).  More generally, the
-   precision is set by the precision of the integration endpoints
-   (promoted to floating-point types).
-
-   The integrand ``f(x)`` can return any numeric scalar, vector, or matrix
-   type, or in fact any type supporting ``+``, ``-``, multiplication
-   by real values, and a ``norm`` (i.e., any normed vector space).
-   Alternatively, a different norm can be specified by passing a `norm`-like
-   function as the `norm` keyword argument (which defaults to `vecnorm`).
```

Comment:
looks like `norm` and `vecnorm` were previously a bit wrong here, should be double-backquoted but are only single